recipes-domu/domu-image-fusion/domu-image-fusion.bbappend

```diff
@@ -1,18 +1,19 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-FILESEXTRAPATHS_prepend := "${THISDIR}/../../inc:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
 
 ###############################################################################
 # extra layers and files to be put after Yocto's do_unpack into inner builder
 ###############################################################################
 # these will be populated into the inner build system on do_unpack_xt_extras
 # N.B. xt_shared_env.inc MUST be listed AFTER meta-xt-prod-extra
```

Comment:
This comment line need to be rewritten, taking into account removal of xt_shared_env.inc.

Reply:
agree